dbt/include/trino/macros/crossdb/listagg.sql

```diff
@@ -0,0 +1,7 @@
+{% macro trino__listagg(measure, delimiter_text, order_by_clause, limit_num) -%}
+    {% set collect_list %} array_agg({{ measure }} {% if order_by_clause -%}{{ order_by_clause }}{%- endif %}) {% endset %}
```

Comment:
@mdesmet  does https://trino.io/docs/current/functions/aggregate.html#listagg fit for this macro?

Reply:
It does not, that's what we had before. However it was not working as per dbt expected behavior and doesn't support all use cases. See https://github.com/starburstdata/dbt-trino-utils/blob/0.1.1/macros/dbt_utils/cross_db_utils/listagg.sql